docs/whats-new/whats-new.md

```diff
@@ -11,6 +11,7 @@ This page summarises the new features, improvements and bug fixes completed in e
   [**View all production release notes**](/whats-new/production-release-notes)
 
 **Staging release notes**
```

Comment:
need to add 1 for 6 oct production release above?

Reply:
Done, waiting for FE and BE version numbers for prod

Reply:
Updated version numbers as they are the same as the staging environment.